sources/metricdog/README.md

```diff
@@ -0,0 +1,55 @@
+# metricdog
+
+Current version: 0.1.0
+
+## Introduction
+
+Metricdog sends anonymous information about the health of a Bottlerocket host.
+It does so by sending key-value pairs as query params in an HTTP GET request.
+
+Metricdog also has the ability to check that a list of critical services is running.
+It does so using `systemctl` and reports services that are not healthy.
+
+## What it Sends
+
+#### The standard set of metrics:
+
+* `sender`: the application sending the report.
+* `event`: the event that invoked the report.
+* `version`: the Bottlerocket version.
+* `variant`: the Bottlerocket variant.
+* `arch`: the machine architecture, e.g.'x86_64' or 'arm'.
```

Comment:
nit: `aarch64`

Reply:
I missed this, need to fix.

Reply:
Fixed in [560f336](https://github.com/bottlerocket-os/bottlerocket/compare/11b09274da45b6642d81f8164f2104b6157026e4..560f33678840b377569ea0dbb2d01633767172ae)